README.rst

```diff
@@ -110,10 +110,10 @@ of the profiles the user will be prompted for his password but on
 subsequent logins the user will not be prompted for his password
 because it has been stored in a secure keyring.
 
-For example when we initially login to prod::
+For example when we initially login to prod:
```

Comment:
Please change the `:` back to `::` this is required by RST for correct formatting. It was not a typo.